src/guide/agent-functions/agent-communication.rst

```diff
@@ -270,10 +401,10 @@ Similar to spatial messaging, array messages provide several iterators for acces
 ================================= =============================================== ==================================
 Iterator                          Usage                                           API Docs
 ================================= =============================================== ==================================
-Moore Neighbourhood               ``FLAMEGPU->message_in(<arguments>)``           :func:`1D<flamegpu::MessageArray::In::operator()>`, :func:`2D<flamegpu::MessageArray2D::In::operator()>`, :func:`3D<flamegpu::MessageArray3D::In::operator()>`
```

Comment:
I understand why you've done this change, but I don't really like it (or have a better suggestion).

Reply:
Have added a separate C++ and Python usage column. Although the API Docs column is unnecessarily large (I think because it is possibly based on the pre-processed URL text rather than the display text). Not a massive issue but a bit annoying.